### ISSUE TITLE
[Bug] [Mac] After importing a .md file with citations or pasting a citation from outside Zettlr, then exporting to PDF, the result does not show citation styling

### ISSUE THREAD

**Issue by zeerph** [Zettlr/Zettlr#3230]
### Description

After importing a .md file with citations that was created in another program, then exported to pdf, citations where expected to be shows properly `(Brown, 2006, p. 206)` but, instead got `(brown_test_2006?)`.

## Testing produced:

* Pasted citations from outside Zettlr and imported ones look like this
* <img width="167" alt="Screen Shot 2022-02-17 at 18 52 49" src="https://user-images.githubusercontent.com/38686030/154462612-f0f437eb-bbcb-4b6d-ab0c-9c6757fbd418.png">

* Tested the same file on my Linux (x86) laptop, Zettlr version 2.2.2, and got the intended result. No errors found.

* The expected results are produced if a CSL file and bibliography are added to the YAML frontmatter, but not to the Zettlr preferences.

* The error on export was found to happen only when a CSL file and bibliography were added to the Zettlr preferences.

* When a reference is typed into Zettlr, citations are functioning as intended in the main screen and on export. This error only appears to happen on importing a file saved outside of Zettlr, or a citation pasted into it.

### Reproducing

* Copy and paste a citation after putting two sets of dashes `---` for YAML (there seems to be an error here, too [Zettlr lets me right click and "open attachment", but nothing appears to happen])
* Need to have a CSL-style and a citation database selected in Zettlr preferences

Paste something like this into Zettlr
```markdown
---

---

Testing citations

[@brown_2006 p. 183]

# References

```

The result will be, after export to PDF
```markdown
Testing citations

(brown_2006?)

# References

Brown, B. 'This is a test'. 2006. 
```

The expected result should be something like (depending on the citation style)

```markdown
Testing citations

(Brown, 2006, p. 183)

# References

Brown, B. 'This is a test'. 2006.
```

### Zettlr Version

Stable (most recent version)

### Specify version

2.2.2

### Your Platform

- [ ] Windows
- [X] macOS
- [ ] Linux

### Architecture

- [ ] Intel 64bit
- [X] ARM 64bit

### Operating System Version

11.6.2

**Comment by nathanlesage** [comment]
So wait, I don't quite follow your importing/opening ... are you importing a file or just opening one outside of the loaded workspaces?

**Comment by zeerph** [comment]
> So wait, I don't quite follow your importing/opening ... are you importing a file or just opening one outside of the loaded workspaces?

Initially the error was discovered after importing a file and loading a workspace.
The same error was also encountered after pasting a citation from outside Zettlr entirely.

The error does also occur after opening a file outside a loaded workspace, but loading or unloading workspaces seems to have no effect on the citation styling. If the citation was typed into Zettlr, it works as intended. If it was not typed into Zettlr, the styling does not appear.

**Comment by nathanlesage** [comment]
Are there any errors logged when this happens? If so, could you please share them?

**Comment by nathanlesage** [comment]
Mh, this is not completely clear to me, but apparently this has nothing to do with the citations themselves, so at least that's out of the woods!

**Comment by zeerph** [comment]
The error seems to have been fixed with update 2.2.4
